### PR TITLE
:sparkles: Working Menu Drawer

### PR DESCRIPTION
**Pull request by Qoyyuum**
Got a working menu drawer but unable to route properly.

### CODE REVIEW

**Review by yash9263 — COMMENTED** [review]
I will try to solve the routing problem

File: src/components/Drawer.jsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      className={clsx(classes.list, {
+        [classes.fullList]: anchor === 'top' || anchor === 'bottom',
+      })}
+      role="presentation"
+      onClick={toggleDrawer(anchor, false)}
+      onKeyDown={toggleDrawer(anchor, false)}
+    >
+      <List>
+        {['users'].map((text, index) => (
+          <Link to={text}>
```
Comment (yash9263) [inline src/components/Drawer.jsx:51]: Link is only used to render a link. If you want to change the directory then you need to add Router. @Qoyyuum

File: src/components/Drawer.jsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+const useStyles = makeStyles({
+  list: {
+    width: 250,
+  },
+  fullList: {
+    width: 'auto',
+  },
+});
+
+export default function AppDrawer() {
```
Comment (yash9263) [inline src/components/Drawer.jsx:26]: Name of the jsx file and default export should be same.
```suggestion
export default function Drawer() {
```

Comment (Qoyyuum) [reply src/components/Drawer.jsx:26]: Ok I'll rename it

File: src/components/Drawer.jsx (hunk 1 above)
Comment (Qoyyuum) [reply src/components/Drawer.jsx]: Fixed in the latest commit

**Comment by Qoyyuum** [comment]
@yash9263 I got the routing working but it needs some improvements so that its easier to maintain and develop when adding new screens/components. Please review and suggest.

**Comment by yash9263** [comment]
I can't understand what its doing. I think you are showing all the users that can use the admin correct me if I am wrong.

**Comment by yash9263** [comment]
I have conflicting files I am not sure what to do with it. If  you @Qoyyuum can help.

**Comment by Qoyyuum** [comment]
I created Users as an example but it's a way to view who the users are (to be later improved when the API is up.)

For the conflict file, you just need to compare and resolve the differences between them. Then pick which ones you want.

**Pull request merged**